Update CODEOWNERS [PIMCORE-145]

atsheehan opened the pull request:
Updating CODEOWNERS to reflect team ownership of this artifact within Salsify.

jturkel reviewed (commented):
I thought we were removing `@salsify/infrastructure-services` from codeowners?

atsheehan committed:
Update CODEOWNERS [PIMCORE-145]

Updating CODEOWNERS to reflect team ownership of this artifact within
Salsify.

atsheehan committed:
Update CI image

The salsify_ci images have been removed, switching to use the ones
[provided by Circle][0].

[0]: https://circleci.com/developer/images/image/cimg/ruby

atsheehan committed:
Mispelled Ruby image
